lib/osf-components/addon/components/search-page/filter-facet/styles.scss

```diff
@@ -1,8 +1,11 @@
 .facet-wrapper {
-    border-top: 1px solid $color-border-gray;
     padding: 0.5rem 0;
 }
 
+.facet-wrapper:not(:first-of-type) {
```

Comment:
I realize we've been using `:first-of-type` inappropriately in our codebase, as `:first-of-type` should be targeting an element type (e.g. `wrapper div:first-of-type` to target the first div). This works and doesn't break things, but it may be worth removing the `:not(:first-of-type)` and removing the border-bottom from the `.left-panel-header`. Will probably need to add a `border-type` to `.no-properties` as well.

Reply:
Thank you for your input. I have tested both scenarios and this works best when there are both filter facets and no filter facets. If you would like to discuss it further, I am happy to do so.